### PR TITLE
Allows to skip embeds on single target #301

### LINKED ISSUE #301: [feature request] Auto reputation on single target
In a conversation like

> questioner: How do I XYZ?

> Answerer: By doing XYZ

> questioner: Thank you [without any mention]

It would be nice if the bot could simply assume that the only answerer should have its reputation incremented

### PR DESCRIPTION
**Pull request by RainbowDashLabs**
Allows to skip the embed when only a single target was identified.

Close #301

### CODE REVIEW

**Review by Goldmensch — APPROVED** [review]

File: src/main/java/de/chojo/repbot/analyzer/ThankType.java [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
+    NO_MATCH;
+
+    private final String localizedName;
+
+    ThankType(String localizedName) {
+        this.localizedName = localizedName;
+    }
+    ThankType() {
+        localizedName = null;
+    }
```
Comment (Taucher2003) [inline src/main/java/de/chojo/repbot/analyzer/ThankType.java:20]: We should stay consistent

You can choose one:

```suggestion
    ThankType(String localizedName) {
        this.localizedName = localizedName;
    }

    ThankType() {
        this.localizedName = null;
    }
```

```suggestion
    ThankType() {
        this(null);
    }

    ThankType(String localizedName) {
        this.localizedName = localizedName;
    }
```

File: src/main/java/de/chojo/repbot/dao/access/guild/settings/sub/Messages.java [hunk 2]
```diff
@@ -138,4 +153,5 @@ ON CONFLICT(guild_id)
                        }).insert()
                        .executeSync() > 0;
     }
+
```
Comment (Taucher2003) [inline src/main/java/de/chojo/repbot/dao/access/guild/settings/sub/Messages.java:156]:
```suggestion
```

**Comment by ItsCubeTime** [comment]
Hi @RainbowDashLabs , thanks for adding the feature. What is the command to enable it?

**Comment by RainbowDashLabs** [comment]
@ItsCubeTime It is currently scheduled for the next release. But at the moment we need to do some more documentation work before we can release the changes, since the current release is the largest feature release we had until now.

You can join our discord to get informed when we release a new version: https://discord.gg/5DrGmz7pHj

**Pull request merged**